src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -75,6 +75,65 @@ public new abstract class Arm64 : ArmBase.Arm64
             ///   A64: RBIT Vd.16B, Vn.16B
             /// </summary>
             public static Vector128<sbyte> ReverseElementBits(Vector128<sbyte> value) { throw new PlatformNotSupportedException(); }
+
+            /// uint8_t vaddv_u8(uint8x8_t a)
+            ///   A64: Bd, Vn.8B
+            /// </summary>
+            public static byte AddAcross(Vector64<byte> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// int8_t vaddv_s8(int8x8_t a)
```

Comment:
I would use the following order for the methods with the same name but different argument types as suggested  by Tanner:

> ... ordering them according to the full type name... so Byte, Double, Int16, Int32, Int64, SByte, Single, UInt16, UInt32, then UInt64

AFAIK this is to make the order consistent with the output of GenApi.dll that we use for generating of reference assemblies. @tannergooding Is it correct?